omnigibson/systems/micro_particle_system.py

```diff
@@ -1444,14 +1443,9 @@ def use_isosurface(self):
     @property
     def particle_radius(self):
         # Magic number from omni tutorials
-        # See https://docs.omniverse.nvidia.com/prod_extensions/prod_extensions/ext_physics.html#offset-autocomputation
+        # See https://docs.omniverse.nvidia.com/extensions/latest/ext_physics/physics-particles.html#offset-autocomputation
         return 0.99 * 0.6 * self._particle_contact_offset
 
-    @property
-    def particle_particle_rest_distance(self):
-        # Magic number, based on intuition from https://docs.omniverse.nvidia.com/extensions/latest/ext_physics/physics-particles.html#particle-particle-interaction
-        return self.particle_radius * 2.0 * m.FLUID_PARTICLE_PARTICLE_DISTANCE_SCALE
```

Comment:
is this not needed anymore?

Reply:
No, I can no longer find any reference to the arbitrary 0.8 scale on their documentation. Empirically, if we keep this 0.8 scale, the sampled particles (e.g. filling a container) will be too close together, which leads to violent explosion when physics starts.